pkg/kubelet/apis/kubeletconfig/types.go

```diff
@@ -222,8 +203,6 @@ type KubeletConfiguration struct {
 	// requests - pull, logs, exec and attach.
 	// +optional
 	RuntimeRequestTimeout metav1.Duration
-	// experimentalMounterPath is the path of mounter binary. Leave empty to use the default mount path
-	ExperimentalMounterPath string
```

Comment:
@jiayingz FYI

Reply:
@jsafrane FYI

Reply:
We (=OpenShift) do not use it. @saad-ali @jingxu97, FYI, you probably need to change your GCI magic a bit.